### ISSUE TITLE
Endless loop if 'sauserprefs_score_inc' not defined

### ISSUE THREAD

**Issue by vel21ripn** [johndoh/roundcube-sauserprefs#48]
Version 1.17.1

Two tabs (general and bayes) use construction:
```
line 509
for ($i = 1; $i <= 10; $i = $i + $rcmail->config->get('sauserprefs_score_inc')) {
...
}
...
line 892
for ($i = 1; $i <= 20; $i = $i + $rcmail->config->get('sauserprefs_score_inc')) {
...
}
```
if 'sauserprefs_score_inc' not defined then endless loop up to exhaust memory.
We must set a minimum value of at least 0.1.

See PR #49

**Comment by johndoh** [comment]
Confirmed. A missing config val should not cause an error like this.

Note: this config option has already been removed in git-master

**Comment by johndoh** [comment]
fixed in release-1.17 and master branches
